scripts/install-prometheus-operator.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+# apply the crds and wait for them
+git clone https://github.com/prometheus-operator/kube-prometheus.git --depth 1 -b v0.11.0
+cd kube-prometheus   
+oc create -f manifests/setup
+until oc get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done
```

Comment:
Why do you need endless loop on the command failure? By the way, if you `set -o errexit` the script will exit on a first failure.

Reply:
I agree. 
I removed the line - "until oc get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done" and added this line - "set -o errexit" in the begining.